nexus/src/app/background/sync_switch_configuration.rs

```diff
@@ -301,7 +300,6 @@ impl BackgroundTask for SwitchPortSettingsManager {
                     },
                 };
 
-                info!(&log, "applying switch port config changes"; "changes" => ?changes);
```

Comment:
Just a high level note that `SwitchPortSettingsManager::activate` should be broken out into smaller functions as it's over 700 lines now.

Reply:
But not in this PR.